server/issue_parser.go

```diff
@@ -28,7 +28,43 @@ func mdKeySummaryLink(issue *jira.Issue) string {
 	return fmt.Sprintf("[%s](%s%s)", issue.Key+": "+issue.Fields.Summary, issue.Self[:pos], "/browse/"+issue.Key)
 }
 
-func parseIssue(issue *jira.Issue) []*model.SlackAttachment {
+func mdTransitionActions(client Client, issue *jira.Issue) ([]*model.PostAction, error) {
```

Comment:
The `md` prefix is meant for functions returning markdown. This is a similar function in that it is returning information to be rendered in the post. @levb thoughts on this?

Reply:
Hello @mickmister , thanks for the review,
> "Looking at the screenshot, I think it makes sense to have a submit button in the post. Does this work along with the dropdown in the Post Action?"

I have observed that clicking on a transition state triggers a request to transition automatically and I was thinking a submit button isn't really necessary.

> "It is unclear what the status of the ticket is in the post. Maybe show the status in the fields above the dropdown?"

I am not sure if the slack attachment can be dynamically changed and maybe I will need some help there. I can also certainly show the issue's status as of the first execution of `/jira view` but after that, unless instead of showing a simple ephemeral post like "*10001 transitioned to Closed*" we show the a post with an attachment again. 

> "Also, does the call to get transitions provide only valid transitions that make sense with the current status?"

I am not sure if there's a way to filter that out at the moment. Like for example if we have "Under review" status that can only transition to "To do"?

Reply:
> I can also certainly show the issue's status as of the first execution of `/jira view`

Yes, this is what I was thinking

> "Also, does the call to get transitions provide only valid transitions that make sense with the current status?"
> > I am not sure if there's a way to filter that out at the moment.

No worries. I was just curious of Jira did it for us. Trying to implement it on our side may be error-prone.

> But what I can do is to filter the actions such that the current issue status is not among them?

Sounds good to me :+1:

Thanks @AGMETEOR!!

Reply:
@mickmister implemented your feedback. I am also still looking for a way to write tests for these changes as the code coverage seems to have slightly fallen.